src/main/webapp/WEB-INF/jsp/fullStory.jsp

```diff
@@ -21,16 +21,130 @@
+                    <select class="news-feeds-container"></select>
+                </c:when>
+                <c:otherwise>
+                    <ul class="news-feeds-container"></ul>
+                </c:otherwise>
+            </c:choose>
+            <br />
+            <div class="titlebar portlet-titlebar">
+                <h3 class="title story-title">${storyTitle}</h3>
+                <a style="text-decoration:none;" href="<portlet:renderURL/>"><img src="<c:url value="/images/arrow_left.png"/>" style="vertical-align: middle;"> <spring:message code="back.list"/></a>
+            </div>
+            <br />
+            <div id="${n}full-story">
+            <c:choose>
+                <c:when test="${ feedView == 'select' }">
+                    ${fullStory}
+                </c:when>
+                <c:otherwise>
+                    <c:forEach var="feed" items="${feeds}">
+                        <c:choose>
+                            <c:when test="${feed.id == activeFeed}">
+                                <div id="${n}feed${feed.id}">${fullStory}</div>
+                            </c:when>
+                            <c:otherwise>
+                                <div id="${n}feed${feed.id}"></div>
+                            </c:otherwise>
+                        </c:choose>
+                    </c:forEach>
+                </c:otherwise>
+            </c:choose>
+            </div>
+        </div>
+        <div class="news-footer">
+            <p>
+                <a style="text-decoration:none;" href="<portlet:renderURL/>"><img src="<c:url value="/images/arrow_left.png"/>" style="vertical-align: middle;"> <spring:message code="back.list"/></a>
+            </p>
+            <p>
+            <c:if test="${supportsHelp}">
+                <a href="<portlet:renderURL portletMode='help'/>"><spring:message code="help" /></a>
+            </c:if>
+            <c:if test="${supportsEdit && !isGuest}">
+                &nbsp;|&nbsp;<a href="<portlet:renderURL portletMode='edit'/>"><spring:message code="edit.news.feed" /></a>
+            </c:if>
+            <c:if test="${isAdmin}">
+                &nbsp;|&nbsp;<a href="<portlet:renderURL portletMode="edit"><portlet:param name="action" value="administration"/></portlet:renderURL>"><spring:message code="administration" /></a>
+            </c:if>
+            </p>
+        </div>
+    </div>
+</div>
+
+<script type="text/template" id="${n}feed-list-template">
+    {{!-- populate either the dropdown options or list items for feeds --}}
+    {{#each this}}
+    <c:choose>
+        <c:when test="${ feedView == 'select' }">
+            <option value="{{id}}">{{name}}</option>
+        </c:when>
+        <c:otherwise>
+            <li id="${n}feed{{id}}-tab"><a href="#${n}feed{{id}}">{{name}}</a></li>
+        </c:otherwise>
+    </c:choose>
+    {{/each}}
+</script>
+
+<script type="text/javascript">
+    var $ = ${n}.jQuery;
```

Comment:
Needs to be placed within a function.  This statement puts jQuery BACK on the global namespace as $.